pkg/logs/logstatus/info.go

```diff
@@ -4,7 +4,7 @@
 // Copyright 2016-present Datadog, Inc.
 
 //nolint:revive // TODO(AML) Fix revive linter
-package status
+package logstatus
```

Comment:
Can we put this code in or in a sub-package of `pkg/logs/status`? 
It's unclear from a quick glance why we need a `pkg/logs/status` and `pkg/logs/logstatus` as they both sound like they do the same thing.

Reply:
Sure, do you want to keep the name `logstatus` for the package? I can move it to `pkg/logs/status/logstatus` if that works.